src/main/java/touro/simondrums/SimonGame.java

```diff
@@ -0,0 +1,29 @@
+package touro.simondrums;
+
+import java.util.*;
+
+public class SimonGame {
+    private ArrayList<Drum> drumSequence = new ArrayList<>();
+
+    public SimonGame(){
+    }
+
+    public boolean checkResponse(Drum userResponse){
+        int index = drumSequence.size()-1;
+        if(userResponse.equals(drumSequence.get(index)))
+        {
+            return true;
+        }
+        else return false;
+    }
+    public ArrayList<Drum> buildSequence(){
```

Comment:
so if I'm understanding this right, this method makes a list of 4 elements, which are randomly selected from the Drum enum?

It could be I'm wrong, but I thought that the number of steps increments by 1 every round, here it would always be 4 (because `drums.size()` will always be the length of the enum).